### ISSUE TITLE
Landing page suggestion

### ISSUE THREAD

**Issue by Bradymck** [district0x/memefactory#143]
Our other apps have various user types and the landing page takes advantage of those types to lead them to the right place of the app on first visit. 

**Ethlance user types:** 
- Become a freelancer
- Find Work
- Become an Employer
- Find Candidates

**Name Bazaar User types:**
- View offerings
- Create Offerings
- How it works
- Trending listings

Considering this design pattern our users have come to know, I think we should run with it and fine tune it even more. 

**Meme Factory has these user types:** 
- Creators
- Curators
- Collectors

Possibly add a Registry button as well. 

A similar landing page may help guide people in a more intuitive way and stay true to previous design patterns.

**Comment by madvas** [comment]
Yeah, this may be added after MVP, but I think a good alternative, if not better, is the UI tutorial we've talked about, that would guide you through all important steps/pages.

**Comment by Bradymck** [comment]
I would agree, not sure this would even be needed. I'll just close this out if we are doing a tutorial.
